packages/vue-components/src/TipBox.vue

```diff
@@ -1,92 +1,108 @@
 <template>
-  <div :class="['alert box-container', containerStyle(), addClass]" :style="customStyle()">
-    <!-- Header wrapper, not rendered if there is no header attribute -->
-    <div v-if="headerBool()" :class="['box-header-wrapper', { 'alert-dismissible': dismissible }]">
+  <div
+    :class="['alert box-container', containerStyle(), addClass]"
+    :style="customStyle()"
+  >
+    <div style="display: flex; flex-direction: row;">
       <!-- icon on the left of the header -->
       <div
-        v-if="iconBool()"
+        v-if="iconBool() && headerBool()"
         :class="['icon-wrapper', iconStyle()]"
         :style="customIconColorStyle()"
       >
         <slot name="icon">
           <i :class="['fas', getFontAwesomeIconStyle()]"></i>
         </slot>
       </div>
-
-      <!-- header -->
-      <div class="box-header">
-        <slot name="_header"></slot>
-      </div>
-
-      <!-- dismiss button to the right of the header -->
-      <button
-        v-if="dismissible"
-        type="button"
-        class="close close-with-heading"
-        data-dismiss="alert"
-        aria-label="Close"
-      >
-        <span aria-hidden="true">&times;</span>
-      </button>
-    </div>
-
-    <!-- Header wrapper -- body wrapper divider for seamless boxes with the header attribute -->
-    <div
-      v-if="horizontalDividerBool()"
-      class="horizontal-divider"
-      :class="getBootstrapAlertStyle()"
-      aria-hidden="true"
-    ></div>
-
-    <!-- Body wrapper -->
-    <div
-      :class="[
-        'box-body-wrapper',
-        {
-          'alert-dismissible': dismissible && !headerBool(),
-          'box-body-wrapper-with-heading': headerBool()
-        }]"
-    >
-      <!-- icon on the left, not shown if there is a header -->
-      <div
-        v-if="iconBool() && !headerBool()"
-        :class="['icon-wrapper', iconStyle()]"
-        :style="customIconColorStyle()"
-      >
-        <slot name="icon">
-          <i :class="['fas', getFontAwesomeIconStyle()]"></i>
-        </slot>
-      </div>
-
       <!-- Icon -- content divider for seamless boxes without the header attribute -->
       <div
-        v-if="verticalDividerBool()"
+        v-if="verticalDividerBoolForHeader()"
         class="vertical-divider"
         :class="getBootstrapAlertStyle()"
         aria-hidden="true"
       ></div>
-
-      <!-- Content wrapper -->
-      <div class="contents" :style="customColorStyle()">
-        <slot></slot>
+      <!-- Header wrapper, not rendered if there is no header attribute -->
```

Comment:
some of the boxes are being shrunk by half 👀 

https://deploy-preview-1415--markbind-master.netlify.app/userguide/usingcomponents#boxes